js/src/builtin/streams/ReadableByteStreamController.cpp

```diff
@@ -761,15 +567,231 @@ enum BYOBRequestSlots {
   unwrappedBYOBRequest->setFixedSlot(BYOBRequestSlot_Controller,
                                      UndefinedValue());
 
-  // Step 3: Set controller.[[byobRequest]].[[view]] to undefined.
+  // // Step 3: Set controller.[[byobRequest]].[[view]] to undefined.
   unwrappedBYOBRequest->setFixedSlot(BYOBRequestSlot_View, UndefinedValue());
 
-  // Step 4: Set controller.[[byobRequest]] to undefined.
-  unwrappedController->clearBYOBRequest();
+  // // Step 4: Set controller.[[byobRequest]] to undefined.
+  unwrappedController->clearByobRequest();
 
   return true;
 }
 
 // Streams spec, 3.13.25.
 //      ReadableByteStreamControllerShouldCallPull ( controller )
 // Unified with 3.10.3 above.
+
+// https://streams.spec.whatwg.org/#abstract-opdef-readablebytestreamcontrollergetbyobrequest
+[[nodiscard]] static bool ReadableByteStreamController_byobRequest(
+    JSContext* cx, unsigned argc, Value* vp) {
+  CallArgs args = CallArgsFromVp(argc, vp);
```

Comment:
I think we might need need to check that the minimum amount of arguments have been passed
```suggestion
  CallArgs args = CallArgsFromVp(argc, vp);
  if (!args.requireAtLeast(cx, "ReadableByteStreamControllerGetBYOBRequest", 1)) {
    return false;
  }
```